src/App.scss

```diff
@@ -0,0 +1,203 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Slab:wght@400;700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@700&display=swap");
+
+:root {
+  //fonts
+  --Monserrat: "Montserrat", sans-serif;
+  --Roboto: "Roboto Slab", serif;
+}
```

Comment:
- [ ] Kudos for using CSS variables to avoid repetition of rules. Great work. 👏

Reply:
Thank you for your comments, it means a lot!